### PR TITLE
Information view changes

### PR DESCRIPTION
**Pull request by dylanlcole**

### CODE REVIEW

File: src/app/components/homePage/admin/allAvailability.js [hunk 1]
```diff
@@ -64,10 +64,10 @@ class AllAvailability extends React.Component {
     Object.keys(jobs).map((jobID) => {
       jobElements.push(
         <div key={jobID} className='job'>
-        <h5>Job Name: {jobs[jobID].job_name}</h5>
-        <h5>Job Date: {jobs[jobID].job_date}</h5>
-        <h5>Job Time: {jobs[jobID].job_time_start + ' ' + jobs[jobID].job_time_end}</h5>
-        <h5>Job Location: {jobs[jobID].location}</h5>
+        <h5 className="col-sm-3">Job Name: {jobs[jobID].job_name}</h5>
```
Comment (Riffmeister) [inline src/app/components/homePage/admin/allAvailability.js:67]: Why are you bringing in bootstrap here?

File: src/app/components/homePage/admin/eventJobs.js [hunk 2]
```diff
@@ -61,22 +62,23 @@ class EventJobs extends React.Component {
...
-                  <h4>Description:</h4>
-                  <h5>{job.jobDescription}</h5>
-                </div>
-                <div>
                   <h4>Job Status:</h4>
                   <h5>{job.jobStatus ? job.jobStatus : 'No Job Status'}</h5>
                 </div>
                 <div>
-                  <h4>Assigned Volunteer:</h4>
+                  <h4>AVolunteer:</h4>
```
Comment (Riffmeister) [inline src/app/components/homePage/admin/eventJobs.js:69]: Spelling

File: src/app/components/homePage/admin/assignPortal.less [hunk 3]
```diff
@@ -90,16 +91,18 @@
 
   .jobs{
     display: flex;
-    flex-shrink: 0;
+    /* flex-shrink: 0; */
```
Comment (Riffmeister) [inline src/app/components/homePage/admin/assignPortal.less:94]: commented code should be removed.

File: src/app/components/homePage/admin/assignPortal.less [hunk 4]
```diff
@@ -90,16 +91,18 @@
...
     flex-grow: 1;
     width: 100%;
+    /* flex-direction: column; */
```
Comment (Riffmeister) [inline src/app/components/homePage/admin/assignPortal.less:97]: Same thing here

File: src/app/components/homePage/admin/allAvailability.js (hunk 1 above)
Comment (dylanlcole) [reply src/app/components/homePage/admin/allAvailability.js:67]: I am doing it for visual purposes, better organize data

**Pull request closed without merging**